packages/api/src/command/medical/patient/convert-fhir-bundle.ts

```diff
@@ -120,16 +121,25 @@ async function convertFHIRBundleToMedicalRecord({
     })
     .promise();
 
+  const isSandbox = Config.isSandbox();
+  const patientMatch = patientMatches(patient.data);
+
   // Send it to conversion
   const payload: ConversionInput = {
     fileName,
     patientId: patient.id,
     firstName: patient.data.firstName,
     cxId: patient.cxId,
+    bucketName:
+      isSandbox && patientMatch
+        ? // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+          Config.getSandboxSeedBucketName()!
+        : Config.getMedicalDocumentsBucketName(),
```

Comment:
I'm thinking we should have the sandbox logic only here on the API, with the FHIR to MR lambda unaware of sandbox at all.

Now that I stop thinking about it, there's no need to call a FHIR to MR lambda just so it calls another lambda. It would if the API didn't know about it, but since we have the seed data on the API and need it to decide the patient match, we could just have a fork in the logic here:
- is it `sandbox` and the patient match? Call the cdaToVisualization lambda w/ the seed data
- otherwise, call the FHIR to MR lambda w/ the FHIR payload (it will convert it and then call the cdaToVis)

Reply:
Yea good call this makes a lot of sense